CHANGELOG.md

```diff
@@ -6,9 +6,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.8.0]
+### Added
+- [#12881](https://github.com/MetaMask/metamask-extension/pull/12881): Feature: Transaction Insights
+  - Users can now see much greater details on contract interaction transactions
+  - These transaction details are integrated with new popups for viewing and editing address book information
+  - These details are also viewable within the transaction details popup
+- [#12627](https://github.com/MetaMask/metamask-extension/pull/12627): Add support for eip-1559 on Trezor
```

Comment:
I believe we should specify the model eip-1559 works on. In my case only the Model T worked and the Model One was still using type-0 txs.

```suggestion
- [#12627](https://github.com/MetaMask/metamask-extension/pull/12627): Add support for eip-1559 on Trezor Model T
```

Reply:
The Trezor Model One will only support EIP-1559 with firmware version 1.10.4, which [Trezor has said will be released mid December (this month)](https://github.com/trezor/trezor-firmware/pull/1861#issuecomment-962282946).

Afterwards, we'd have to wait until `trezor-connect` (JS library) is updated, then update `Metamask/eth-trezor-keychain` before there will be Trezor Model One support.